taccsite_cms/settings.py

```diff
@@ -268,6 +268,14 @@ def gettext(s): return s
 
 
 
+########################
+# TACC: CORE STYLES
+########################
+
+# Only use integer numbers (not "v1", not "0.11.0"),
+# so templates can load based on simple comparisons
+TACC_CORE_STYLES_VERSION = 0
+
```

Comment:
Eventually, will we be able to drop the if blocks for the version check, once all the portals are migrated forward? Or do you want to keep this setting to future-proof v3 development.

If retained, is the v1/v2 filename nomenclature the presence of the `site.` prefix in the v1 filenames (eg. 'site_cms/css/build/site.app.blog.css' vs 'site_cms/css/build/app.blog.css')?

Reply:
<details><summary><strong>Uncertain</strong>, the future of this setting is.</summary>

I want to remove it after migration, but we may be slow to migrate. Proof: CMD **still** has **CMS v1** (pre Core-Styles) CSS like A2CPS and Frontera.

</details>

<details><summary><strong>Yes</strong>, the <code>site.</code> prefix distinguishes CMS styles that use Core-Styles v0.</summary>

I do not like this, because it is not obvious. I think these `site.____.css` stylesheets belong in https://github.com/TACC/Core-CMS/tree/main/taccsite_cms/static/site_cms/css/src/_migrations. But Core-Portal and Frontera Tech Docs expect the filenames `site.header.css` and `site.tacc-search-bar.css`, and I have been too lazy to work in those repos again.

</details>